### PR TITLE
feat: Add Balanced Parenthesis class

### PR DESCRIPTION
**Pull request by RogerMo01**

### CODE REVIEW

File: OOP-Exercises/BalancedParenthesis.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OOP_Exercises
+{
+    class BalancedParenthesis
```
Comment (LongJohnSilver1504) [inline OOP-Exercises/BalancedParenthesis.cs:9]: Good, make it an static class..... same comment as previuos classs

Comment (RogerMo01) [reply OOP-Exercises/BalancedParenthesis.cs:9]: Ok

Comment (RogerMo01) [reply OOP-Exercises/BalancedParenthesis.cs:9]: Done!

**Review by LongJohnSilver1504 — APPROVED** [review]
LGTM

**Comment by LongJohnSilver1504** [comment]
Solve conflict here

**Pull request merged**